src/ax_interface/mib.py

```diff
@@ -127,10 +135,16 @@ def __init__(self, subtree, sub_ids, value_type, callable_, *args, updater=None)
 
     def __iter__(self):
         for sub_id in self.sub_ids:
-            yield self.sub_id + (sub_id,)
+            if isinstance(sub_id, tuple):
```

Comment:
Created a new MIBEntry subclass. Considering the dynamic OID feature, do some minor changes on existing CME class to facility the upper level implementation.

---
In reply to: [97460638](https://github.com/Azure/sonic-snmpagent/pull/3#discussion_r97460638) [](ancestors = 97460638)